judge/management/commands/create_problem.py

```diff
@@ -1,6 +1,7 @@
 from django.core.management.base import BaseCommand
 
 from judge.models import Problem, ProblemGroup, ProblemType
+from judge.models.runtime import Language
```

Comment:
I think we should be able to import `Language` directly from `judge.models` (i.e. merge the import on this line with the previous line)

Reply:
Thanks for pointing it out. It is more concise.